graphql/me/fields.ts

```diff
@@ -3,7 +3,19 @@ import { getSessionUser, GraphQLUser } from '../authentication';
 import { GraphQLContext } from '../context';
 import { Role } from '../authorizations';
 import { UserType } from '../types/user';
+import { Field, ObjectType } from 'type-graphql';
 
+@ObjectType()
+class Contact {
```

Comment:
Can't we use the User (UserType) here?

Reply:
I think that the `Contact` type would maybe extended by fields e.g. `contactReason`, if we add a own `contact` table.

Reply:
Well how about:

class Contact {
  user: UserType;
  contactReason: ...;
  chatId: string;
}